src/Pages/sangyeonkim/Login/Login.js

```diff
@@ -1,14 +1,113 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./Login.scss";
+import { FaFacebookSquare } from "react-icons/fa";
 
 class Loginkim extends Component {
-	state = {  }
-	render() { 
-		return (
-			<div>
-
-			</div>
-		);
-	}
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+    };
+  }
+
+  goToMain = (e) => {
+    e.preventDefault();
+    if (this.state.id.includes("@") && this.state.pw.length > 4) {
+      fetch("http://10.58.4.128:8000/user/signin", {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.id,
+          password: this.state.pw,
+        }),
+      })
+        .then((response) => response.json())
+        .then((result) => console.log("결과: ", result));
+      this.props.history.push("./Main-sangyeonkim");
+    }
+  };
+
+  handleIdInput = (event) => {
+    this.setState({ id: event.target.value });
+  };
+
+  handlePwInput = (event) => {
+    this.setState({ pw: event.target.value });
+  };
+
+  render() {
+    const { id, pw } = this.state;
+    return (
+      <>
+        <section className="container">
+          <section className="structure">
+            <div className="instagram">
+              <p className="westaTitle">Westagrams</p>
+            </div>
+            <div className="loginPage">
+              <div>
+                <input
+                  type="text"
+                  className="id"
+                  placeholder="전화번호, 사용자 이름 또는 이메일"
+                  onChange={this.handleIdInput}
+                />
+              </div>
+              <div>
+                <input
+                  type="password"
+                  className="pw"
+                  placeholder="비밀번호"
+                  onChange={this.handlePwInput}
+                />
+              </div>
```

Comment:
바로 코드 적용만 하시면 다음에 계산된 속성명을 활용할 때 또 헷갈리실 수 있습니다.
공부를 꼭 해보시고! 적용해주세요 :)

```suggestion
              <div>
                <input
                  type="text"
                  className="id"
                  name="id" // 계산된 속성명을 사용하려면 이부분과 state 명을 같게 해주어야 겠죠?
                  placeholder="전화번호, 사용자 이름 또는 이메일"
                  onChange={this.handleIdInput}
                />
              </div>
              <div>
                <input
                  type="password"
                  className="pw"
                  name="pw" // 계산된 속성명을 사용하려면 이부분과 state 명을 같게 해주어야 겠죠?
                  placeholder="비밀번호"
                  onChange={this.handlePwInput}
                />
              </div>
```